payment/datastore.go

```diff
@@ -21,15 +25,71 @@ import (
 	_ "github.com/golang-migrate/migrate/v4/source/file"
 )
 
+// Methods type is a string slice holding payments
+type Methods []string
```

Comment:
Nit: If we renamed Methods to PaymentMethods, then we wouldn't need the comment above and it makes grepping easier long term

Reply:
linting will fail for stuttering.  payment.PaymentMethods.